service/server.go

```diff
@@ -127,6 +128,15 @@ func (s *server) ListenAndServe(ctx context.Context) chan []error {
 	wg := new(sync.WaitGroup)
 	wg.Add(2)
 
+	currentUser, err := user.Current()
+	if err != nil {
+		glg.Warn(err)
```

Comment:
IMO: unhandled errors

Should we handle the error?
```go
		err := glg.Warn(err)
		if err != nil {
			glg.Fatal(errors.Wrap(err, "xxx error log output failed"))
			return nil
		}	
```

Reply:
@hiyosi 
No, because the Garm application doesn't need to rely on `user.Current()` to run as explained in https://github.com/AthenZ/garm/pull/41#discussion_r1818367787

Reply:
I see.  I totally understand.

Reply:
The purpose of this pr is to print out the user and its ID, not to make the garm stricter by requiring `user.Current()`. This change is more of an enhancement or a robustness improvement.